content/learning-paths/smartphones-and-mobile/function-multiversioning/examples.md

```diff
@@ -0,0 +1,417 @@
+---
+title: Examples
+weight: 3
+
+### FIXED, DO NOT MODIFY
+layout: learningpathall
+---
+
+## Examples
+
+#### Code Generation example
+
+In this example we have specified two versions of `foo` using the `target_clones` attribute (the order in which they are listed does not matter). At certain optimization levels compilers can decide to perform loop vectorization depending on the target's vector capabilities. Our intention is to enable the compiler to use SVE instructions in the specialized case, whilst restricting it to use only Armv8 instructions in the default one.
+
+loop.c
+```c
+__attribute__((target_clones("sve", "default")))
+int foo(int *v, unsigned n) {
```

Comment:
nit: maybe `foo` should have a more meaningful name. `sumPosEltsScaledByIndex` or similar, as names like `foo` can be inaccessible and hard to grok for some people.